### ISSUE TITLE
Some questions about training on my dataset.

### ISSUE THREAD

**Issue by Yzhongming** [orrzohar/PROB#38]
Hello, I have trained 4 tasks on my own dataset. The mAP scores are increasing from Task 1 to Task 4, with Task 1 being 45.3 and Task 4 exceeding 80. I want to know how to set the number of epochs for each task. Should I train until over 80 for Task 1, which may require a large number of epochs? Another question is about the batch size. Due to batch size reasons, I set the learning rate to 8e-5. I noticed that you changed the learning rate to 2e-5 for Task 2, 3, and 4. How should I adjust it appropriately?

**Comment by orrzohar** [comment]
Hi @Yzhongming,

Could you share the evaluation curves (mAP over epochs)?

Difference in mAP between the different stages could be due to you having a much smaller dataset, and therefore requireing more epochs to saturate or due to the relative difficulty of each task. You usually want to see mAP begin saturating before switching to the next task.

Best,
Orr

**Comment by Yzhongming** [comment]
Thank you for your response. I did not save the mAP curve, but I have noticed that it has not converged yet before starting other tasks. I am currently making adjustments.
